mqtt-transport/src/main/java/com/esri/geoevent/transport/mqtt/MqttInboundTransport.java

```diff
@@ -74,16 +65,19 @@ public void start() throws RunningException
 		{
 			switch (getRunningState())
 			{
-				case STARTING:
-				case STARTED:
-				case STOPPING:
-					return;
+			case STARTING:
+			case STARTED:
+			case STOPPING:
+				return;
+			}
+			if (log.isTraceEnabled())
+			{
+				log.trace("Starting MQTT Inbound Transport");
 			}
 			setRunningState(RunningState.STARTING);
 			thread = new Thread(this);
 			thread.start();
-		}
-		catch (Exception e)
+		} catch (Exception e)
```

Comment:
formatting - the `catch` statement needs to start a new line:
```suggestion
		} catch (Exception e)
```
```suggestion
		} catch (Exception e)
```
````
try
{
  ...
}
  catch (...)
{
  ...
}
```